test/memspaces/memspace_highest_bandwidth.cpp

```diff
@@ -30,7 +34,10 @@ static bool canQueryBandwidth(size_t nodeId) {
                                   numaNode, &initiator, 0, &value);
 
     hwloc_topology_destroy(topology);
-    return (ret == 0);
+
+    if (ret != 0) {
+        GTEST_SKIP() << "ret is equal to " << ret << ", should be " << 0;
```

Comment:
can we make these messages a little more meaningful? this `ret` is bound specifically to the result of `hwloc_memattr_get_value`